geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,82 @@
+# Geschäftsordnung der Vollversammlung der Fachschaft Digital Engineering
+
+Diese Geschäftsordnung wurde am xx.xx.xxxx von der Vollversammlung beschlossen.
+
+## § 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für die Vollversammlung der Fachschaft Digital Engineering an der Universität Potsdam.
+
+## § 2 Tagesordnung
+
+(1) Die Vollversammlung folgt einer Tagesordnung.
+
+(2) Die Tagesordnung muss Teil der fristgerechten Einladung einer Vollversammlung sein.
+
+(3) Bis 14 Tage vor einer Vollversammlung können die einladenden Personen gestellte Anträge in die Tagesordnung aufnehmen.
+
+(4) Bis 7 Tage vor einer Vollversammlung muss die finale Tagesordnung auf dem gleichen Weg wie die Einladung zur Vollversammlung bekanntgegeben werden.
+
+## § 3 Versammlungsleitung
+
+(1) Die Vollversammlung wird von den einladenden Personen geleitet.
+
+(2) Die Versammlungsleitung kann für einzelne Tagesordnungspunkte an andere Mitglieder der Fachschaft übergeben werden.
+
+(3) Die Versammlungsleitung kann bei Bedarf eine Redeliste führen und/oder die Redezeit begrenzen.
+
+(4) Die Versammlungsleitung entscheidet, auf welche Weise Stimmen gezählt oder geschätzt und ob sie anschließend protokolliert werden.
+
+## § 4 Protokolle
+
+(1) Die Vollversammlung dokumentiert ihre Beschlüsse in einem Protokoll.
+
+(2) Die Versammlungsleitung beauftragt ein Mitglied der Fachschaft mit der Protokollführung.
+
+(3) Das Protokoll wird der Fachschaft spätestens 7 Tage nach der Vollversammlung zugänglich gemacht.
+
+## § 5 Anträge
+
+(1) Anträge sind schriftlich zu verfassen und innerhalb von 14 Tagen nach Zugang der Einladung zur Vollversammlung bei den einladenden Personen einzureichen.
+
+(2) Die einladenden Personen entscheiden über die Aufnahme des Antrags in die Tagesordnung.
+
+(3) Anträge können vor der Behandlung durch die antragstellende Person zurückgezogen werden.
+
+## § 6 Änderungsanträge
+
+(1) Änderungsanträge werden vor der Abstimmung des jeweiligen Antrags behandelt.
+
+(2) Änderungsanträge können von der den ursprünglichen Antrag stellenden Person direkt angenommen werden. Geschieht dies nicht, stimmt die Vollversammlung über die Annahme ab.
```

Comment:
Mögliches Problem: A stellt einen Antrag, B schlägt Änderung vor, die mit großer Mehrheit angenommen wird, aber von A nicht gemocht wird. A beauftragt jetzt ihren/seinen Freund/in C damit, einen Änderungsantrag zu stellen, der dann von A sofort angenommen wird. Damit können zwei Leute (Antragsteller/in und eine weitere Person) den Mehrheitswillen der Fachschaft überstimmen. 

Das wird zwar vermutlich in der Regel dazu führen, dass der Antrag einfach nicht angenommen wird. Aber manche vielleicht ist die Änderung nur total "unwesentlich" und sollte nicht zur Ablehnung eines ansonsten guten und wichtigen Antrags führen, v.a. wenn dieser ja wegen der Antragsfristen auch nicht einfach neu durch eine andere Person gestellt werden kann.

Reply:
Dann würde eine Sitzungsleitung in deinem Beispiel einfach einen entsprechenden neuen Änderungsantrag einbringen, der den Willen der Mehrheit abbildet. Außerdem würden A und C gemaßregelt werden.
Ich sehe hier kein Schadenspotential, nur Potential von A und C, zu nerven und sich unbeliebt zu machen.

Im Gegensatz dazu gibt es z.B. uns als FSR die Möglichkeit, eingehende Änderungsanträge direkt anzunehmen, wenn wir sie gut finden, statt sie erst auf der Sitzung behandeln zu müssen